R/settings.R

```diff
@@ -50,18 +39,45 @@ read_settings <- function(filename) {
   }
 }
 
+read_config_file <- function(config_file) {
+  if (is.null(config_file)) {
+    return(NULL)
+  }
+
+  if (endsWith(config_file, ".R")) {
+    config <- new.env()
+    read <- function(file) {
+      sys.source(file, config)
+      config
+    }
+    malformed <- function(e) {
+      stop("Malformed config file, ensure it is valid R syntax\n  ", conditionMessage(e), call. = FALSE)
+    }
+  } else {
+    read <- function(file) read.dcf(file, all = TRUE)
```

Comment:
This is where the `get_setting()` akwardness should go IMO

Reply:
Yep, that's what I was thinking. Will do.